plugins/events-node/src/api/EventsService.ts

```diff
@@ -55,3 +55,8 @@ export type EventsServiceSubscribeOptions = {
  * @public
  */
 export type EventsServiceEventHandler = (params: EventParams) => Promise<void>;
+
+/**
+ * @public
+ */
+export const EVENTS_NOTIFY_TIMEOUT_HEADER = 'x-backstage-events-notify-timeout';
```

Comment:
Not sure if there's already some convention in this project for custom Backstage headers - let me know if I should change this

Reply:
ah, only thing is to not use the x- prefix as that's generally discouraged nowadays

Reply:
ah ok cool - thanks for informing me, was not aware of the updated standards